python/pyspark/sql/connect/dataframe.py

```diff
@@ -2306,7 +2183,7 @@ def _test() -> None:
     )
 
     (failure_count, test_count) = doctest.testmod(
-        pyspark.sql.connect.dataframe,
+        pyspark.sql.dataframe,
```

Comment:
It does inherit the docstrings but `doctest` cannot. So I manually switch it to `pyspark.sql.dataframe` to run the doctest.